### DISCUSSION TITLE
Sensor install issue

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#13779] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

32

### Storage for /

1000

### Storage for /nsm

18000

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hello, 
I am adding a first sensor to a new grid. I am getting the following error upon completion of the install of a sensor
****Installation does not appear to be complete. A highstate has not fully completed.**** 

Please advise

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by robbiemarshall** [comment]
Are you able to run a so-checkin?  Are any errors produced?  Have you checked the sosetup.log in /root for errors?
